cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+    "\n",
+    "In a true multiannotator setting this would not be possible since ground truth labels will not exist. Instead we have [benchmarked](link paper or blogpost maybe) this method to ensure this method produces more accurate `consensus_labels` than common inustry methods for getting a consensus."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3758a5fb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import and load ground truth labels (once again this is only for metrics, normally not have this information)\n",
+    "!cd $experiment_path && wget -nc 'https://cleanlab-public.s3.amazonaws.com/Multiannotator/cifar-10h/cifar-10h-worst25-coin20/c10h_test_labels.npy'\n",
+    "\n",
+    "true_labels = np.load(f'{experiment_path}/c10h_test_labels.npy')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "06915543",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Calculate CROWDLAB performance against the ground truth for each epoch\n",
+    "for i in range(itter):\n",
+    "    pred_probs = model_results[i]['pred_probs']\n",
+    "    consensus_labels_in = model_results[i]['consensus_labels_in']\n",
+    "    consensus_labels_out = model_results[i]['consensus_labels_out']\n",
+    "\n",
+    "    acc_model_gtruth = (pred_probs.argmax(axis=1) == true_labels).mean()\n",
+    "    acc_consensus_gtruth = (consensus_labels_in == true_labels).mean()\n",
+    "\n",
+    "    results = {\n",
+    "        \"consensus_gtruth_accuracy\": acc_consensus_gtruth, # consensus labels accuracy \n",
+    "        \"model_gtruth_accuracy\": acc_model_gtruth,         # model label accuracy\n",
+    "    }\n",
+    "    \n",
+    "    model_results[i].update(results)\n",
+    "\n",
+    "# Calculate accuracy of final CROWDLAB generated consensus label\n",
+    "final_consensus_labels = model_results[itter-1][\"consensus_labels_out\"]\n",
+    "final_consensus_accuracy = (final_consensus_labels == true_labels).mean()\n",
+    "\n",
+    "print(\"Final consensus label accuracy (vs ground truth): \", final_consensus_accuracy)\n",
+    "print(\"Final model predictions accuracy (vs ground truth): \", model_results[itter-1][\"model_gtruth_accuracy\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03bb6b74",
+   "metadata": {},
+   "source": [
+    "Finally lets plot the results over multiple epochs and observe the improvement. Here epoch -1 corresponds to the initial `consensus_labels` generated using majority vote without the help of a model. Generating consensus with majority vote is the most common practice and therefore makes a great baseline to compare CROWDLAB against.\n",
+    "\n",
+    "As you can see, the accuracy of the `consensus_labels` improves over several itterations of model training."
```

Comment:
Be careful about using a phrase like "iterations of model training". This can be easily confused with something like epochs in one fit() call of an iteratively-trained model, whereas here we are really referring to repeated rounds of: 

training model on current consensus labels and obtaining probabilistic predictions from model, using these predictions to improve the current consensus labels.

Reply:
using rounds!

Reply:
@huiwengoh You make a great point about cannonically using itter (I did that originally too!) but here's discussion about why rounds might be better.

Reply:
Ah did not see this. Makes sense - thanks for clarification!